### ISSUE TITLE
Feature request: polygon mask option

### ISSUE THREAD

**Issue by mjgleason** [GeoBigData/nacho#5]
I'd like to be able to run nacho with an optional polygon mask that limits the areas where we would expect positive occurrences of features to occur. For example, if I was interested in finding crosswalks, I might want to feed in a polygon geojson representing buffered roads. Nacho would then output all of the chips that intersect that polygon into one folder (e.g., `positive_candidates`) and all chips that don't intersect to another folder (e.g., `negatives`).

**Comment by drwelby** [comment]
Yes, this would be a good feature, even if just simply implemented as a mask with an in/out option
